packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -4817,7 +4826,7 @@ export type Resources = {
   // usedImagePreloads: Set<PreloadResource>,
   precedences: Map<string, Set<StyleResource>>,
   stylePrecedences: Map<string, StyleTagResource>,
-  usedStylesheets: Set<PreloadResource>,
```

Comment:
We recomputed the key before using props however with variance on props shape this was going to be even more expensive. Turns out we could just store the key with the resource by making this change which I should have done all along